ravem/indico_ravem/util.py

```diff
@@ -112,16 +107,28 @@ def has_access(event_vc_room, _split_re=re.compile(r'[\s,;]+')):
     event = event_vc_room.event
     current_user = session.user
 
-    # No physical room or room is not Vidyo capable
-    if not room or not room.has_equipment('Vidyo'):
+    # No physical room or room is not videoconference capable
+    feature = RavemPlugin.settings.get('room_feature')
```

Comment:
with the suggestions from above, this will already return a `RoomFeature` instance, so no need to query it manually